compat/os/mswindows.h

```diff
@@ -639,6 +642,7 @@ inet_pton(int af, const void *src, char *dst)
 #endif
```

Comment:
You have added (a very helpful) `HAVE_DECL_INET_NTOP` comment to the `else` clause in inet_ntop(). I cannot attach a change request to the corresponding inet_pton() function, but it should have a similar comment (mentioning `HAVE_DECL_INET_PTON`).

Reply:
Addressed in d6716bc0b4a3045cf2997fe540a85454b0c6a9b2 under the EasierDone agreement.